mithril.js

```diff
@@ -1959,9 +1988,17 @@
 		}
 	}
 
+	function isNativeError(e) {
```

Comment:
I replaced that comment, because I wasn't thinking.

As for the change, it's going to be pulled out of this one.